events/diskpersist.go

```diff
@@ -72,14 +73,16 @@ type DiskPersistOptions struct {
 	DIDCacheSize    int
 	EventsPerFile   int64
 	WriteBufferSize int
+	Retention       time.Duration
 }
 
 func DefaultDiskPersistOptions() *DiskPersistOptions {
 	return &DiskPersistOptions{
-		EventsPerFile:   10000,
-		UIDCacheSize:    100000,
-		DIDCacheSize:    100000,
+		EventsPerFile:   20,
```

Comment:
20 seems wrong

Reply:
Yeah, good catch we tweaked it while making sure garbage collection worked, bumping it back to 20k

Reply:
err, 10k oops